pkg/terraform/stages/nutanix/stages.go

```diff
@@ -0,0 +1,13 @@
+package nutanix
+
+import (
+	"github.com/openshift/installer/pkg/terraform"
+	"github.com/openshift/installer/pkg/terraform/providers"
+	"github.com/openshift/installer/pkg/terraform/stages"
+)
+
+// PlatformStages are the stages to run to provision the infrastructure in Nutanix.
+var PlatformStages = []terraform.Stage{
+	stages.NewStage("nutanix", "pre-bootstrap", []providers.Provider{providers.Nutanix}),
```

Comment:
Can we call this stage "cluster" instead of "pre-bootstrap"? The resources for the cluster are created by the stage.

Reply:
Sure.